src/collections/_documentation/platforms/php/index.md

```diff
@@ -108,11 +108,12 @@ $builder->setTransport($transport);
 
 Hub::getCurrent()->bindClient($builder->getClient());
 ```
-{% capture __note %}
-> Note: `Hub::getCurrent()->bindClient($builder->getClient());` is required to tell the global function calls to use your new `Client` instead.
-{% endcapture %}
 
-{{ __note }}
+{% include components/alert.html
```

Comment:
I think the `{% capture __note %}` must stay since I use it multiple times on this page.
Here for example: https://github.com/getsentry/sentry-docs/pull/726/files#diff-710202863a09fce49d82f164296a394eR139

Reply:
@HazAT nice catch! Yes, let's keep the Notes. However, to stay consistent with all the Notes in docs (this is a relatively new Best Practice thing), I've put them all in an alert box. I just pushed the changes.